packages/rocketchat-reactions/setReaction.js

```diff
@@ -1,7 +1,12 @@
 /* globals msgStream */
 Meteor.methods({
-	setReaction(reaction, messageId) {
-		if (!Meteor.userId()) {
+	setReaction(reaction, messageId, user) {
```

Comment:
It was previously assumed that a reaction was coming from the current user.  This is true most of the time, but in my case when Slack sends a reaction, I needed to call this method on behalf of the user.  So added a user parameter, but if not present, it behaves as before.